pkg/alertmanager/operator.go

```diff
@@ -143,7 +145,17 @@ func New(c prometheusoperator.Config, logger log.Logger) (*Operator, error) {
 }
 
 func (c *Operator) RegisterMetrics(r prometheus.Registerer) {
-	r.MustRegister(NewAlertmanagerCollector(c.alrtInf.GetStore()))
+	c.statefulsetErrors = prometheus.NewCounter(prometheus.CounterOpts{
+		Namespace: "prometheus_operator",
```

Comment:
namespace and subsystem are subject to removal in the client lib, I'd suggest to not start using it